src/tests/auth/logout.test.js

```diff
@@ -0,0 +1,22 @@
+import chaiHttp from 'chai-http';
+import chai, { expect } from 'chai';
+import app from '../../app';
+import { loggedInToken, createUsers } from '../fixtures/users.fixture';
+
+chai.use(chaiHttp);
+describe('Tests for user login', () => {
+  before(async () => {
+    await createUsers();
+  });
+  it('Should logout a user', (done) => {
```

Comment:
Also, add a test case to login using the logged out token to be sure the user is really logged out

Reply:
Ok

Reply:
I just added a test on not allowing the user to access the protected routes when signed out. 
For what you said above, we are not logging in using the token. instead, we generate token after login. we check token then when accessing protected routes.